src/Microsoft.ML.Vision/ImageClassificationTrainer.cs

```diff
@@ -769,7 +769,7 @@ private static Tensor EncodeByteAsString(VBuffer<byte> buffer)
             unsafe
             {
                 fixed (byte* src = buffer.GetValues())
-                    c_api.TF_StringEncode(src, (UIntPtr)length, (sbyte*)(tensor + sizeof(Int64)), size, status);
+                    c_api.TF_StringEncode(src, (ulong)length, (byte*)(tensor + sizeof(Int64)), size, status.Handle);
```

Comment:
@codemzs  Why the pointer needs to +8 here

Reply:
Tensorflow uses first 8 bytes to save string length.